concent_api/core/payments/backends/sci_backend.py

```diff
@@ -51,10 +55,46 @@ def get_list_of_payments(
             from_block      = last_block_before_payment,
             to_block        = payment_interface.get_block_number(),  # pylint: disable=no-member
         )
-
     return payments_list
 
 
+def validate_that_last_closure_time_is_older_than_last_payment(
+    requestor_eth_address: str,
+    provider_eth_address: str,
+    youngest_payment_ts: Optional[int],
+) -> None:
+    if youngest_payment_ts is None:
+        return
+    forced_payment_event_list = _get_list_of_forced_payment_events(
+        requestor_eth_address=requestor_eth_address,
+        provider_eth_address=provider_eth_address,
+        search_payments_since_ts=youngest_payment_ts,
+    )
+    for forced_payment_event in forced_payment_event_list:
+        if youngest_payment_ts < forced_payment_event.closure_time:
+            raise BanksterTimestampError
+
+
+def get_youngest_payment_timestamp_from_subtask_results_accepted_list(subtask_results_accepted_list: list) -> Optional[int]:
+    return max(subtask_results_accepted.payment_ts for subtask_results_accepted in subtask_results_accepted_list)
+
+
+def _get_list_of_forced_payment_events(
```

Comment:
This function does exactly the same as `get_list_of_payments` called with `transaction_type=TransactionType.FORCE`.